### DISCUSSION TITLE
Windows 11 + ExplorerPatcher is the best

### DISCUSSION THREAD

**Discussion by gtamborero** [valinet/ExplorerPatcher#2282] [General]
Just after updating (now) Windows 11, explorer patcher have disappeard and I installed it again. 
I have been a few minutes without explorer patcher and the win11 explorer is so disgusting...
Thank you very much Valinet!!
Donating now :)

**Comment by bubbismith** [comment]
Totally agree. MS makes changes for changes sake. ExplorerPatcher is the best 'go back to previous version' software I've used. I still use Open-Shell to get rid of the Win11 menu, and various other utilities since Windows XP. But EP offers the best features of them all. Thanks so much.

**Comment by xMcSqueezy** [comment]
Agree. I peronally dislike the Win11 menu start and love the tiles in Win10, so when I found out about this program, I immediately got it and even added some Win11 aesthetics to it, such as rounded corners, since I don't mind that style, infact it looks pretty good. Totally a must-have for me since I upgraded to 11.

**Comment by Clyde257** [comment]
Just wanted to say I appreciate everything the creators are doing. I've always loved the Windows 10 Start Screen and Never Really Liked the New Box. Reminds me of chromebooks. I really appreciate the recent update that allows me to pin stuff to the start screen now. Keep it up Devs 👍🏽
